pom.xml

```diff
@@ -143,7 +150,11 @@
             <artifactId>jersey-common</artifactId>
             <version>2.22.2</version>
         </dependency>
-
+        <dependency>
```

Comment:
Please add a comment about what this is for.

Reply:
Updated.